go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/ava-labs/avalanche-cli
 
-go 1.22
-
-toolchain go1.22.5
+go 1.22.5
```

Comment:
Just double checking the release versions, we typically use the latest patch version of the minor release one behind latest.

This is because we want to include all of the security patches that have been published, and want to give the latest minor version time to become more stable.

In this case, we should be on `1.22.7`
https://go.dev/doc/devel/release#go1.22.minor